tests/pocket_ic_test.py

```diff
@@ -34,9 +34,8 @@ def test_create_canister_with_id(self):
 
         # Creating a canister with an ID that is not hosted by any subnet fails.
         canister_id = ic.Principal.anonymous()
-        with self.assertRaises(ValueError) as ex:
+        with self.assertRaises(Exception) as ex:
```

Comment:
you are not using `ex` anymore?

Reply:
Yes, the error returned has changed, so I figured the test does not need to be so strict

Reply:
removed the variable

Reply:
the test has been refactored